RadarSDK/RadarAPIClient.m

```diff
@@ -496,4 +497,39 @@ - (void)ipGeocodeWithCompletionHandler:(RadarIPGeocodeAPICompletionHandler)compl
     }];
 }
 
+- (void)getContextWithLocation:(CLLocation * _Nonnull)location
+             completionHandler:(RadarGetContextAPICompletionHandler _Nullable)completionHandler {
+    NSString *publishableKey = [RadarSettings publishableKey];
+    if (!publishableKey) {
+        return completionHandler(RadarStatusErrorPublishableKey, nil, nil);
+    }
+
+    NSMutableString *queryString = [NSMutableString new];
+    [queryString appendFormat:@"coordinates=%.15f,%.15f", location.coordinate.latitude, location.coordinate.longitude];
+
+    NSString *host = [RadarSettings host];
+    NSString *url = [NSString stringWithFormat:@"%@/v1/context?%@", host, queryString];
+    
+    NSDictionary *headers = @{
+        @"Authorization": publishableKey,
+        @"Content-Type": @"application/json",
+        @"User-Agent": [RadarUtils userAgent],
+        @"X-Radar-Config": @"true",
+    };
+    
+    [self.apiHelper requestWithMethod:@"GET" url:url headers:headers params:nil completionHandler:^(RadarStatus status, NSDictionary * _Nullable res) {
+        if (status != RadarStatusSuccess || !res) {
+            return completionHandler(status, nil, nil);
+        }
+        
+        id contextObj = res[@"context"];
+        RadarContext *context = [[RadarContext alloc]  initWithObject:contextObj];
+        if (context) {
+            return completionHandler(RadarStatusSuccess, res, context);
+        }
+
+        completionHandler(RadarStatusErrorServer, nil, nil);
```

Comment:
hmm, so when the server does return 200 but the result does not include context, we just return a nil result? what's the usage of other fields within 'res'?

Reply:
if the result doesn't include the `context` key, then the above block won't get executed, and we'll throw a 500 - I think the point is that we always expect `context` to be a top-level key in the response for this route